devito/types/grid.py

```diff
@@ -786,10 +775,22 @@ def __subdomain_finalize__(self, grid, counter=0, **kwargs):
                 sd_func.data[:, idx] = self._local_bounds[idx]
 
             dname = '%si%d' % (d.name, counter)
-            dimensions.append(MultiSubDimension(dname, d,
-                                                functions=sd_func,
-                                                bounds_indices=(2*i, 2*i+1),
-                                                implicit_dimension=i_dim))
+
+            ltkn = Symbol(name="%s_ltkn%d" % (d.name, counter), dtype=np.int32,
```

Comment:
NOTE: this is at the moment redundant (see changes to /ir/equations/algorithms above) but it will soon be dropped by a subsequent PR